src/Exporter.php

```diff
@@ -18,6 +18,10 @@ class Exporter implements ExporterInterface
 </graphml>
 EOL;
 
+    /**
+     * @param \Fhaculty\Graph\Graph $graph
```

Comment:
Is there any convention to use fully qualified class names in docblocks? Otherwise, I'd rather use (much shorter) local class names instead. (see also below)